types/events.go

```diff
@@ -79,8 +78,8 @@ func (a Attribute) String() string {
 }
 
 // ToKVPair converts an Attribute object into a Tendermint key/value pair.
-func (a Attribute) ToKVPair() tmkv.Pair {
-	return tmkv.Pair{Key: toBytes(a.Key), Value: toBytes(a.Value)}
+func (a Attribute) ToKVPair() abci.EventAttribute {
```

Comment:
shouldn't this be using the cosmos kv instead of returning an `abci.EventAttribute`? the func name and comment differs with behavior with this update